CIScripts/Test/Utils/IisTcpTestDockerImageBuild.ps1

```diff
@@ -0,0 +1,18 @@
+. $PSScriptRoot\..\..\Common\Aliases.ps1
+$DockerfilePath = "$PSScriptRoot\..\..\DockerImages\iis-tcptest\Dockerfile"
+function Initialize-IisTcpTestDockerImage  {
+    Param ([Parameter(Mandatory = $true)] [PSSessionT] $Session)
+
+    $DockerImagesDir = "C:\DockerImages"
```

Comment:
This directory is actually going to contain dockerfile (not docker image).
Better name could also mention that it is a "remote" location.